Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ruby:2.7.5-slim
+
+ARG bundlerWithout="development test lint"
+ARG yarnFlags="--production"
+
+# Install required software:
+#  - build-essential: to have a compiling environment for building gems
+#  - curl: for setting Node version and healthcheck
+#  - git is a rails gems dependency
+RUN apt-get update
+RUN apt-get install -y build-essential
+RUN apt-get install -y curl
+RUN apt-get install -y git
+
+# Set Node to install version 14
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
+RUN apt-get update
+RUN apt-get install -y nodejs
+
+# Change the working directory for all proceeding operations
+#   https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#workdir
+WORKDIR /code
+
+# Install build tools
+RUN npm install --global yarn
+RUN gem install bundler
+
+# Install bundler based dependencies
+COPY Gemfile .
+COPY Gemfile.lock .
+RUN bundle config set without "${bundlerWithout}"
+RUN bundle install
+
+# Install yarn based dependencies
+COPY package.json .
+COPY yarn.lock .
+RUN rm -rf node_modules && yarn install --frozen-lockfile ${yarnFlags}
```

Comment:
Did you have to add the removal for any reason? At this moment of the build the folder does not have the node_modules folder yet

Reply:
It was move of a "just in case".  It is unnecessary according to the current Dockerfile but it also doesn't do any harm.

Reply:
Ok